sdk/src/main/java/com/sygic/travel/sdk/StSDK.kt

```diff
@@ -1,184 +1,40 @@
 package com.sygic.travel.sdk
 
-import android.arch.persistence.room.Room
 import android.content.Context
 import com.github.salomonbrys.kodein.Kodein
 import com.github.salomonbrys.kodein.instance
-import com.sygic.travel.sdk.api.StApi
-import com.sygic.travel.sdk.db.StDb
 import com.sygic.travel.sdk.di.KodeinSetup
-import com.sygic.travel.sdk.model.media.Medium
-import com.sygic.travel.sdk.model.place.Place
-import com.sygic.travel.sdk.model.place.Tour
-import com.sygic.travel.sdk.model.query.PlacesQuery
-import com.sygic.travel.sdk.model.query.ToursQuery
-import com.sygic.travel.sdk.provider.DataProvider
-import com.sygic.travel.sdk.utils.runAsync
-import com.sygic.travel.sdk.utils.runWithCallback
+import com.sygic.travel.sdk.favorites.facade.FavouritesFacade
+import com.sygic.travel.sdk.places.facade.PlacesFacade
+import com.sygic.travel.sdk.tours.facade.ToursFacade
 
 /**
  * Provides public methods for requesting API.
  */
-class StSDK internal constructor(context: Context) {
+class StSDK internal constructor() {
 	companion object {
-		private val DATABASE_NAME = "st-sdk-db"
-		private lateinit var placesKodeinSetup: Kodein
+		private lateinit var kodeinSetup: Kodein
 
 		/**
 		 * Creates the SDK.
 		 * @param xApiKey Api key - must be provided.
 		 * @param context Application's context.
 		 */
 		fun create(xApiKey: String, context: Context): StSDK {
-			placesKodeinSetup = KodeinSetup.setupKodein(context, xApiKey)
-			return StSDK(context)
+			kodeinSetup = KodeinSetup.setupKodein(context, xApiKey)
```

Comment:
kodeint should be initialized in Sdk, not in static factory.

Reply:
:heavy_check_mark: